src/components/atoms/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -49,20 +50,15 @@ export const ConfirmationModal: React.FC<ConfirmationModalProps> = ({
   const isShown = show !== undefined ? show : isVisible;
 
   return (
-    <Modal
-      className="ConfirmationModal"
-      show={isShown}
-      onHide={hide}
-      centered={centered}
-    >
+    <Modal show={isShown} onHide={hide} centered={isCentered}>
       <Modal.Body>
-        <div>
-          {hasHeader && <Modal.Title>{header}</Modal.Title>}
+        <div className="ConfirmationModal">
```

Comment:
why the class of the ConfirmationModal is moved? it should be set on the container element.

Reply:
it's still a container class for the confirmation content body

Reply:
well, let's say we want to change the background of the confirmation modal. If we change it for the current ".ConfirmationModal" selector, it'll look this way, leaving black borders (see the screenshot). But if we assign this class name to the Modal component, we'll be able to target a child to change the background of the needed element.

![image](https://user-images.githubusercontent.com/63313732/134685606-0e65928f-9bf7-4ebf-bf98-003f55c2b279.png)